club/templates/search.html

```diff
@@ -1,47 +1,13 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group">
-	<!-- wp:search /-->
+<!-- wp:group {"layout":{"inherit":false}} -->
+<div class="wp-block-group"><!-- wp:search /-->
```

Comment:
Design suggest to use a placeholder instead of label and a different text for the button. The search from 404 page could be re-used by extracting to a hidden pattern.

Design:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/1935113/183568278-7780cf51-c3dc-44ad-a1d6-06aa0ae927fa.png">

frontend:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/1935113/183568391-c246cfba-8464-40bc-ae0d-03ce4cf6d87b.png">

Reply:
Thanks for your input. Now it is looking like this:

![image](https://user-images.githubusercontent.com/1310626/183610506-15933d03-8ae6-452c-9b95-d9ce42d1b657.png)